packages/wouter/src/index.js

```diff
@@ -265,7 +265,9 @@ export const Link = forwardRef((props, ref) => {
 
   // handle nested routers and absolute paths
   const href = router.hrefs(
-    targetPath[0] === "~" ? targetPath.slice(1) : router.base + targetPath,
+    (targetPath ?? "")[0] === "~"
```

Comment:
Thank you for the PR!
As a suggestion, this could also be done by providing the default value for `to` when destructuring props:

```js
  const {
    to = "",
    href: targetPath = to,
```
(above in this file)